app/renderer/components/Session/AdvancedServerParams.js

```diff
@@ -9,19 +9,21 @@ export default class AdvancedServerParams extends Component {
 
 
   render () {
-    const {server, setServerParam, t} = this.props;
+    const {server, setServerParam, t, serverType} = this.props;
 
     return <Row gutter={8}>
       <Col className={styles.advancedSettingsContainerCol}>
         <div className={styles.advancedSettingsContainer}>
           <Collapse bordered={true}>
             <Panel header={t('Advanced Settings')}>
               <Row>
+                {
+                  serverType !== "lambdatest" && 
                 <Col span={6}>
                   <FormItem>
                     <Checkbox checked={!!server.advanced.allowUnauthorized} onChange={(e) => setServerParam('allowUnauthorized', e.target.checked, 'advanced')}>{t('allowUnauthorizedCerts')}</Checkbox>
                   </FormItem>
-                </Col>
+                </Col> }
```

Comment:
Could you adjust the indentation?

Reply:
this is done in the latest commit, thank you .